_pages/en_US/installing-boot9strap-(frogtool).txt

```diff
@@ -0,0 +1,88 @@
+{: .notice--warning}
+
+### What You Need
+
+* Your `movable.sed` file from completing Seedminer or nimhax on the SD card root
+* <i class="fa fa-magnet" aria-hidden="true" title="This is a magnet link. Use a torrent client to download the file."></i> [frogcert.bin](magnet:?xt=urn:btih:d12278ea50bb3574f1fbd327f3d0e2292c70941f&dn=frogcert.bin&tr=https%3a%2f%2ftracker.fastdownload.xyz%3a443%2fannounce&tr=https%3a%2f%2fopentracker.xyz%3a443%2fannounce&tr=http%3a%2f%2fopen.trackerlist.xyz%3a80%2fannounce&tr=http%3a%2f%2ft.nyaatracker.com%3a80%2fannounce&tr=udp%3a%2f%2ftracker.tiny-vps.com%3a6969%2fannounce&tr=udp%3a%2f%2fopen.demonii.si%3a1337%2fannounce&tr=udp%3a%2f%2ftracker.port443.xyz%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.vanitycore.co%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.torrent.eu.org%3a451%2fannounce&tr=udp%3a%2f%2fretracker.lanta-net.ru%3a2710%2fannounce&tr=udp%3a%2f%2fthetracker.org%3a80%2fannounce&tr=http%3a%2f%2ftorrent.nwps.ws%3a80%2fannounce&tr=udp%3a%2f%2ftracker.coppersurfer.tk%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.iamhansen.xyz%3a2000%2fannounce&tr=udp%3a%2f%2fbt.xxx-tracker.com%3a2710%2fannounce&tr=http%3a%2f%2f0d.kebhana.mx%3a443%2fannounce&tr=udp%3a%2f%2fexodus.desync.com%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.opentrackr.org%3a1337%2fannounce&tr=udp%3a%2f%2ftracker4.itzmx.com%3a2710%2fannounce&tr=udp%3a%2f%2ftracker.justseed.it%3a1337%2fannounce&tr=http%3a%2f%2ftherightsize.net%3a1337%2fannounce&tr=udp%3a%2f%2fretracker.hotplug.ru%3a2710%2fannounce&tr=udp%3a%2f%2ftracker.internetwarriors.net%3a1337%2fannounce&tr=udp%3a%2f%2f9.rarbg.com%3a2800%2fannounce&tr=https%3a%2f%2f2.track.ga%3a443%2fannounce&tr=udp%3a%2f%2fbigfoot1942.sektori.org%3a6969%2fannounce) (magnet link)
+* The latest release of [Frogtool](https://github.com/zoogie/Frogtool/releases/latest)
+* The latest release of [b9sTool](https://github.com/zoogie/b9sTool/releases/latest)
+
+#### Section I - CFW Check
+
+{% include_relative include/cfw-check-fredtool.txt %}
+
+#### Section II - Prep Work
+
+1. Power off your device
+1. Insert your SD card into your computer
+1. Copy boot.firm and boot.3dsx from the Luma3DS .zip to the root of your SD card
+1. Copy boot.nds (b9sTool) from the b9sTool release .zip to the root of your SD card
+1. Copy Frogtool.3dsx to the /3ds/ folder on your SD card
+1. Copy frogcert.bin to the root of your SD card
+1. Reinsert your SD card into your device
+1. Power on your device
+
+#### Section III - Patching DS Download Play
+
+1. Open the Homebrew Launcher through your method of choice (likely the Internet Browser)
+1. Launch Frogtool from the list of homebrew
+1. Select the "INJECT patched DS Download Play" option
+1. Frogtool will automatically run and inject the JPN version of Flipnote Studio into your DS Download Play
+1. Once this operation has finished, read the screens and check if the process was successful
+1. If there are any errors or missing files, correct the problem and try again
```

Comment:
"correct the problem" is not very clear, maybe just link to troubleshooting if this is absolutely necessary (which it might not be)